x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.tsx

```diff
@@ -259,6 +261,7 @@ const EventsViewerComponent: React.FC<Props> = ({
                       </HeaderFilterGroupWrapper>
                     )}
                   </HeaderSection>
+                  {exceptionsModal && exceptionsModal(refetch)}
```

Comment:
This is the callback that the modal needed access to. I followed the pattern used for the utility bar here.